client/hotrod-client/src/main/java/org/infinispan/client/hotrod/RemoteCacheManager.java

```diff
@@ -507,7 +507,7 @@ private Properties loadFromStream(InputStream stream) {
                log.tracef("Enabling near-caching for cache '%s'", cacheName);
             }
             return new InvalidatedNearRemoteCache<>(this, cacheName, timeService,
-                  createNearCacheService(cacheName, configuration.nearCache()));
```

Comment:
Sorry I even had fixed this in https://github.com/infinispan/infinispan/pull/8216/files#diff-f586b9c1e982f7928d2e729db7b63a39R505 but it was disregarded for awhile. I totally forgot it would be in 10.1.x, so I am glad you made a separate PR :)